packages/blockchain-common/package.json

```diff
@@ -0,0 +1,153 @@
+{
+    "name": "ibm-blockchain-platform-common",
```

Comment:
Probably worth stripping out all of the key/vals apart from the ones we actually need, as it's just a duplicate of `blockchain-extension`. Also, it's not going to be published to npm either.

Are we able to move the license checker and anything else to root level package.json also?

Reply:
The licence checker needs to be in each package